.github/workflows/ci_cd.yml

```diff
@@ -128,8 +128,15 @@ jobs:
 
       - name: Start API, ingest and index test data
         run: just init
+
       - name: Run API tests
+        continue-on-error: true
         run: just api-test -T
+
+      - name: Print API test failure logs
+        if: {{ failure() }}
+        run: just logs
```

Comment:
Should this stop as well, since the previous step failed?

Reply:
I was going to try debugging whether you have to explicitly `exit 1` or something or if the `if: {{ failure() }}` runs just the "failure" jobs and ignores the rest.